static/test/javascripts/spec/common/identity/Api.spec.js

```diff
@@ -34,8 +34,8 @@ define([
                 };
                                getCookieStub = sinon.stub();
                                getCookieStub.withArgs('GU_U').returns(
-                                   'WyIyMzEwOTU5IiwiamdvcnJpZUBnbWFpbC5jb20iLCJqYW1lc2dvcnJpZSIsIjUzNCIsMTM4Mjk1MzAzMT' +
-                                   'U5MSwxXQ.MC0CFBsFwIEITO91EGONK4puyO2ZgGQcAhUAqRa7PVDCoAjrbnJNYYvMFec4fAY'
+                                   'WyIyMzEwOTU5IiwiamdvcnJpZUBnbWFpbC5jb20iLCJBbcOpbGllIErDtHNlIiwiNTM0IiwxMzgyOTUzMDMxNTkxLDFd' +
```

Comment:
@jbreckmckye: Yeah, I should add a note on that. I just wanted to make sure that the fallback is doing the right thing as well (which it does already, but without any tests verifying that).